src/Orchard.Web/Modules/Orchard.AuditTrail/Views/AuditTrailEvent-Content.SummaryAdmin.cshtml

```diff
@@ -1,37 +1,34 @@
-@using Orchard.AuditTrail.Helpers
+﻿@using Orchard.AuditTrail.Helpers
 @using Orchard.AuditTrail.Services.Models
 @using Orchard.ContentManagement
 @{
     var descriptor = (AuditTrailEventDescriptor)Model.Descriptor;
     var eventData = (IDictionary<string, object>) Model.EventData;
     var eventPastTense = descriptor.Name.Text.ToLower();
-    var contentItemId = (int?)Model.ContentItemId;
+    var contentItemId = (int)Model.ContentItemId;
```

Comment:
Please don't change that and the `.Value` after. It was because some old instances have corrupted data with null values.